### ISSUE TITLE
Error when starting the plugin

### ISSUE THREAD

**Issue by Sayrix** [Butzlabben/missilewars#88]
**Describe the bug**
When the server starts, there are an error and the plugin can't be enabled

**To Reproduce**
1. Paper 411 (1.17.1)
2. Java 17J9 (ghcr.io/pterodactyl/yolks:java_17J9)
3. Start the server

**Error:**
```
[14:10:56 INFO]: [MissileWars] Enabling MissileWars v4.5.1
[14:10:56 INFO]: [MW] Boot | This server is running MissileWars v4.5.1 by Butzlabben
[14:10:56 INFO]: [MW] Boot | Loading properties...
[14:10:56 INFO]: [MW] Boot | Registering commands
[14:10:56 INFO]: [MissileWars] [ACF] Enabled Asynchronous Tab Completion Support!
[14:10:56 ERROR]: Error occurred while enabling MissileWars v4.5.1 (Is it up to date?)
java.lang.NoSuchMethodError: com/fasterxml/jackson/dataformat/yaml/YAMLFactory._createContentReference(Ljava/lang/Object;)Lcom/fasterxml/jackson/core/io/ContentReference; (loaded from file:/home/container/plugins/MissileWars-4.5.1.jar by PluginClassLoader{plugin=MissileWars v4.5.1, pluginEnabled=true, url=plugins/MissileWars-4.5.1.jar}) called from class com.fasterxml.jackson.dataformat.yaml.YAMLFactory (loaded from file:/home/container/plugins/MissileWars-4.5.1.jar by PluginClassLoader{plugin=MissileWars v4.5.1, pluginEnabled=true, url=plugins/MissileWars-4.5.1.jar}).
        at com.fasterxml.jackson.dataformat.yaml.YAMLFactory.createParser(YAMLFactory.java:412) ~[MissileWars-4.5.1.jar:?]
        at com.fasterxml.jackson.dataformat.yaml.YAMLFactory.createParser(YAMLFactory.java:386) ~[MissileWars-4.5.1.jar:?]
        at com.fasterxml.jackson.dataformat.yaml.YAMLFactory.createParser(YAMLFactory.java:15) ~[MissileWars-4.5.1.jar:?]
        at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3548) ~[MissileWars-4.5.1.jar:?]
        at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3516) ~[MissileWars-4.5.1.jar:?]
        at de.butzlabben.missilewars.util.serialization.Serializer.yamlToJson(Serializer.java:120) ~[MissileWars-4.5.1.jar:?]
        at de.butzlabben.missilewars.util.serialization.Serializer.deserialize(Serializer.java:68) ~[MissileWars-4.5.1.jar:?]
        at de.butzlabben.missilewars.game.Arenas.load(Arenas.java:68) ~[MissileWars-4.5.1.jar:?]
        at de.butzlabben.missilewars.MissileWars.onEnable(MissileWars.java:103) ~[MissileWars-4.5.1.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[patched_1.17.1.jar:git-Paper-411]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[patched_1.17.1.jar:git-Paper-411]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:500) ~[patched_1.17.1.jar:git-Paper-411]
        at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugin(CraftServer.java:561) ~[patched_1.17.1.jar:git-Paper-411]
        at org.bukkit.craftbukkit.v1_17_R1.CraftServer.enablePlugins(CraftServer.java:475) ~[patched_1.17.1.jar:git-Paper-411]
        at net.minecraft.server.MinecraftServer.loadWorld(MinecraftServer.java:733) ~[patched_1.17.1.jar:git-Paper-411]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:317) ~[patched_1.17.1.jar:git-Paper-411]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1220) ~[patched_1.17.1.jar:git-Paper-411]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319) ~[patched_1.17.1.jar:git-Paper-411]
        at java.lang.Thread.run(Thread.java:857) ~[?:?]
[14:10:56 INFO]: [MissileWars] Disabling MissileWars v4.5.1
[14:10:56 INFO]: [MW] Error | [MySQL] Connection does not exist or was already closed
```

**Comment by RedstoneFuture** [comment]
I think this was already fixed with the last commit.

**Comment by Sayrix** [comment]
Well as you can see i use the 4.5.1

**Comment by RedstoneFuture** [comment]
https://github.com/daniel-naegele/missilewars/commit/3e7a57139ffde284cc14876aa0de26d9a8f78bc7

**Comment by Sayrix** [comment]
This commit is in the 4.5.1?

**Comment by RedstoneFuture** [comment]
No. Wait for the next release.

**Comment by Sayrix** [comment]
Ok! Can you keep the issue opened until the next release?

**Comment by daniel-naegele** [comment]
Should be fixed with 4.5.2
